### PR TITLE
Feat/propagate font manager changes

### LINKED ISSUE #187: Propagate changes from the Font Manager to the Typography Manager
Testing the issue #177 I found that we can improve the new system by auto-changing the font family once you change font value.

### Steps to reproduce
- Add new font to the Font Manager
- Create new Typography in the Typography Manager and select the font family to be the font created before.
- Change the font family in the Font Manager

### Expected result
- To see the font family applied in the customizer

### Current result
- We have to re-select the font-family for the typography

### Definition of done
- [ ] Propagate font change
- [ ] Remove a font once is selected (avoiding duplicates)

### PR DESCRIPTION
**Pull request by JeanPaiva**
This PR is changing a couple of behaviors on the font manager.

- [x] Removing "System default" from Fonts manager list
- [x] Removing selected fonts from the list, to avoid duplicates
- [x] Propagate changes to typography manager
- [x] Add dist files

closes #187

### CODE REVIEW

File: src/customizer-controls/font-manager/GeneratePressFontManagerControlForm.js [hunk 1]
```diff
@@ -26,6 +26,20 @@ const GeneratePressFontManagerControlForm = ( props ) => {
 		wp.customize.control( props.customizerSetting.id ).setting.set( value );
 	};
 
+	const propagateChanges = (currentFontFamily, previousFontFamily) => {
```
Comment (JeanPaiva) [inline src/customizer-controls/font-manager/GeneratePressFontManagerControlForm.js:29]: Not a big fan of this implementation. Ideally, a pub/sub system would be better. Not sure if worth the implementation tho.

Comment (tomusborne) [reply src/customizer-controls/font-manager/GeneratePressFontManagerControlForm.js:29]: I don't mind it - nice and simple.

**Review by tomusborne — APPROVED** [review]
- [x] System default is gone from Font Manager
- [x] Can't add the same font to font manager twice
- [x] Changing a font in the font manager changes it in the typography manager as well

Looks and works great!

Pushed some linting changes so my editor doesn't highlight a bunch of red issues 🙂

**Comment by tomusborne** [comment]
Nice - like that a lot. This looks good to me!

**Pull request merged**